Helper/Order.php

```diff
@@ -1224,7 +1224,7 @@ public function updateOrderPayment($order, $transaction = null, $reference = nul
                 break;
 
             case self::TS_REJECTED_REVERSIBLE:
-                $orderState = OrderModel::STATE_HOLDED;
+                $orderState = OrderModel::STATE_PAYMENT_REVIEW;
```

Comment:
How will the merchant know that the payment approve / reject action needs to take place?
How will this orders be easily distinguishable?

Reply:
@danac-gs this uses default Magento functionality. The merchant will see two buttons on the admin's View Order page https://screencast.com/t/1H8v1ee6d6G8.

Reply:
@johnmccombs1 awesome. thanks for the explanation.